test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -212,7 +211,7 @@ def forward(self, x, k):
     def test_layer_norm(self):
         model = torch.nn.LayerNorm([10, 10])
         x = torch.randn(20, 5, 10, 10)
-        self.run_test(model, x, rtol=1e-05, atol=1e-07)
```

Comment:
Why change this test?

Reply:
CI test sometimes gives test error:
Max absolute difference: 9.536743e-07
Max relative difference: 0.0001676
it is above this specific tolerances. Use default tolerances is stable.